app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+require 'pry'
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/new'
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets/new' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else 
+      @user = current_user #sets @user to session[:user_id]
+      @tweet = Tweet.create(content: params[:content], user: @user) #creates a new Tweet using params and @user and sets it to @tweet
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.